activity.py

```diff
@@ -519,15 +518,12 @@ def read_file(self, file_path):
 
 class ChartData(Gtk.TreeView):
 
-    __gsignals__ = {
-             'label-changed': (GObject.SignalFlags.RUN_FIRST, None, [str, str], ),
-             'value-changed': (GObject.SignalFlags.RUN_FIRST, None, [str, str], ), }
+    __gsignals__ = {'label-changed': (GObject.SignalFlags.RUN_FIRST, None, [str, str], ), 'value-changed': (GObject.SignalFlags.RUN_FIRST, None, [str, str], ), }
```

Comment:
Split it into multiple lines. Seems like this was extra diff. The old code was equally good.

Reply:
Done in 8181469